main.js

```diff
@@ -1,11 +1,25 @@
+let formBody = [
```

Comment:
these must be created using JQuery. You should finish task 2 in order to understand better.